### PR TITLE
Add pulumi

### PR DESCRIPTION
**Pull request by zklaus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xhochy** [comment]
You need to ship the licenses of the bundled static dependencies using `go-licenses`.

File: recipes/pulumi-sdks/build_pulumi_core.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export PULUMI_VERSION=${PKG_VERSION}
+export PULUMI_ROOT=${PREFIX}
+
+make build
+make install
+
+chmod -R u+w ${GOPATH}
+
+go-licenses save . --save-path=../license-file --ignore github.com/pulumi/pulumi
```
Comment (zklaus) [inline recipes/pulumi-sdks/build_pulumi_core.sh:16]:
```suggestion
go-licenses save . --save_path=../license-file --ignore github.com/pulumi/pulumi
```

File: recipes/pulumi-sdks/build_pulumi_go.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export PULUMI_VERSION=${PKG_VERSION}
+export PULUMI_ROOT=${PREFIX}
+
+make -C sdk/go build
+make -C sdk/go install
+
+chmod -R u+w ${GOPATH}
+
+go-licenses save . --save-path=../license-file --ignore github.com/pulumi/pulumi
```
Comment (zklaus) [inline recipes/pulumi-sdks/build_pulumi_go.sh:16]:
```suggestion
go-licenses save . --save_path=../license-file --ignore github.com/pulumi/pulumi
```

File: recipes/pulumi-sdks/build_pulumi_nodejs.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export PULUMI_VERSION=${PKG_VERSION}
+export PULUMI_ROOT=${PREFIX}
+
+make -C sdk/nodejs build
+make -C sdk/nodejs install
+
+chmod -R u+w ${GOPATH}
+
+go-licenses save . --save-path=../license-file --ignore github.com/pulumi/pulumi
```
Comment (zklaus) [inline recipes/pulumi-sdks/build_pulumi_nodejs.sh:16]:
```suggestion
go-licenses save . --save_path=../license-file --ignore github.com/pulumi/pulumi
```

File: recipes/pulumi-sdks/build_pulumi_python.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+export PULUMI_VERSION=${PKG_VERSION}
+export PULUMI_ROOT=${PREFIX}
+
+make -C sdk/python build
+make -C sdk/python install
+
+chmod -R u+w ${GOPATH}
+
+go-licenses save . --save-path=../license-file --ignore github.com/pulumi/pulumi
```
Comment (zklaus) [inline recipes/pulumi-sdks/build_pulumi_python.sh:16]:
```suggestion
go-licenses save . --save_path=../license-file --ignore github.com/pulumi/pulumi
```

**Comment by xhochy** [comment]
You probably need to `cd pkg` before running go-licenses.

**Comment by zklaus** [comment]
Thanks for the help, @xhochy! I think I figured it out now. Could you have another look?

**Review by xhochy — APPROVED** [review]

**Pull request merged**